### DISCUSSION TITLE
Blood bag weights

### DISCUSSION THREAD

**Discussion by Tirpitz93** [acemod/ACE3#8164] [General]
Is it intentional that blood weights about half as much as it should?
For example a 1000ml bag should weight approximately 1kg, but in the arsenal it appears as 1lb, 0.45kg.
Was this done intentionally, if so for what reason?

**Comment by jonpas** [comment]
Arma does not know "weight" or "volume" separately, it knows "mass", which combines the two. It is simply a matter of balance.

**Reply by Tirpitz93** [reply]
This is what I suspected.
